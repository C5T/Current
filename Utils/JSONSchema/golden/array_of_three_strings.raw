{"ObjectOrArray":{"fields":{"":{"String":{"values":{"a":2,"b":1},"instances":3,"nulls":0}}},"instances":1,"nulls":0}}