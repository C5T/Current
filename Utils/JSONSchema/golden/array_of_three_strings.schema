// Autogenerated schema inferred from input JSON data.

using Schema = std::vector<std::string>;
