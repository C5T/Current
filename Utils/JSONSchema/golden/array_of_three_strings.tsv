Field	Type	Set	Unset/Null	Values	Details
Schema	Array	1	0
Schema[]	String	3	0	2 distinct values	`a` : 2, `b` : 1
